packages/wonder-blocks-badge/src/components/badge.tsx

```diff
@@ -165,8 +165,13 @@ const styles = StyleSheet.create({
         color: badgeTokens.root.color.foreground,
     },
     icon: {
-        width: badgeTokens.icon.sizing.width,
-        height: badgeTokens.icon.sizing.height,
+        // Use minWidth and minHeight to ensure custom sized icons don't
+        // overflow the badge
```

Comment:
This is more for if PhosphorIcon or Icon components aren't used with Badge (as a backup)